editor/editor/mode/text.js

```diff
@@ -0,0 +1,9 @@
+function Text( { html, onChange } ) {
+	return (
+		<div className="editor-mode-text">
```

Comment:
Not much, but it eases styling and I'm certain we'll have other elements added soon.